examples/raw.rs

```diff
@@ -114,3 +112,64 @@ fn main() -> Result<()> {
     // Cleanly exit.
     Ok(())
 }
+
+fn parse_args() -> (Vec<String>, Option<(PathBuf, PathBuf, PathBuf)>) {
```

Comment:
it is better to define a struct so we can add more fields later.

Reply:
I thought it was unlikely to have more fields in this simple example. Now, I think defining a struct  may help with the furthur command line tool work, so I adopt your suggestion.

With a struct, I considered using [structopt](https://github.com/TeXitoi/structopt) instead of bare clap, but I still want the raw and transactional example to have different app name. This prevents me from reusing code in the two examples.